### DISCUSSION TITLE
Shall a new release be tagged soon?

### DISCUSSION THREAD

**Discussion by zserge** [golangci/golangci-lint#4121] [Q&A] [answered]
The previous release was almost 2 months ago, and since then some versions of the linters have been bumped (i.e. revive) with some new features included, so perhaps it's time to tag a new patch release these days to make use of them?

**Comment by ldez** [comment, marked as answer]
I have some PRs to review, I will create a release after that.

---

_Sponsoring is a good way to sustain open source maintainers: [sponsor me](https://github.com/sponsors/ldez)_

**Comment by millotp** [comment]
Hey, any update on a release date ?
